src/acquisition/covidcast/nancodes.py

```diff
@@ -0,0 +1,19 @@
+"""Provides unified not-a-number codes for the indicators.
+
+Currently requires a manual sync between the covidcast-indicators
+and the delphi-epidata repo.
+* in covidcast-indicators: _delphi_utils_python/delphi_utils
+* in delphi-epidata: src/acquisition/covidcast
```

Comment:
is there a reason we're not importing these from delphi_utils directly?

Reply:
there is a reason, but it is not a good one (i forgot!) 

fixing now (thanks for catching!).